src/app/containers/ATIAnalytics/atiUrl/index.js

```diff
@@ -183,6 +187,12 @@ export const buildATIPageTrackPath = ({
       value: isLocServeCookieSet(),
       wrap: true,
     },
+    {
```

Comment:
@jimjohnsonrollings do we always want to include the `xto` key even if the request is not from a marketing campaign?

Reply:
I think it's probably best for it not to be there if there's no value for it

Reply:
[see comment](https://github.com/bbc/simorgh/pull/7484#discussion_r471479817)